spec/models/manageiq/providers/vmware/infra_manager/refresher_storage_profile_spec.rb

```diff
@@ -0,0 +1,107 @@
+require Rails.root.join('spec/tools/vim_data/vim_data_test_helper')
+
+describe ManageIQ::Providers::Vmware::InfraManager::Refresher do
+  before(:each) do
+    _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
+    @ems = FactoryGirl.create(
+      :ems_vmware_with_authentication,
+      :zone => zone, :name => "VC41Test-Prod",
+      :hostname => "VC41Test-Prod.MIQTEST.LOCAL",
+      :ipaddress => "192.168.252.14"
+    )
```

Comment:
Being picky, I'd just move these to helpers above this `before` as well:

```
let!(:zone) { EvmSpecHelper.create_guid_miq_server_zone[2] }
let!(:ems) do
      @ems = FactoryGirl.create(
        :ems_vmware_with_authentication,
        :zone => zone, :name => "VC41Test-Prod",
        :hostname => "VC41Test-Prod.MIQTEST.LOCAL",
        :ipaddress => "192.168.252.14"
     )
end
```

(You then need to change all `@ems` to `ems` below)